Tests/Fixtures/NodeJSKitTest/.gitignore

```diff
@@ -7,3 +7,4 @@ DerivedData/
 .swiftpm/config/registries.json
 .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
 .netrc
+.vscode
```

Comment:
Just a small dev experience change to avoid including `launch.json` included, didn't know where else to squeeze this small change in.

Reply:
`~/.config/git/ignore` or `.git/info/exclude`? but it's ok to include it here.